workspaces/src/network/mainnet.rs

```diff
@@ -0,0 +1,90 @@
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use async_trait::async_trait;
+use url::Url;
+
+use near_primitives::views::FinalExecutionStatus;
+
+use crate::network::Info;
+use crate::network::{
+    Account, CallExecution, CallExecutionDetails, NetworkClient, NetworkInfo,
+    TopLevelAccountCreator,
+};
+use crate::rpc::{client::Client, tool};
+use crate::types::{AccountId, InMemorySigner, SecretKey};
+use crate::Contract;
+
+const RPC_URL: &str = "https://rpc.mainnet.near.org";
```

Comment:
Could/Should be env variable with fallback?

Reply:
Could be useful, but won't make it in this PR. Can you setup a new issue detailing it? Not sure what the best design to fit this in, but could be some form of configuration management down the line

Reply:
yeah good call, ill do that